src/main/webapp/scheduled-interviews.js

```diff
@@ -12,6 +12,89 @@
+    .then(response => response.json())
+    .then((scheduledInterviews) => {
+      if (!isEmptyObject(scheduledInterviews)) {
+        document.getElementById('no-scheduled-interviews').style.display = 'none'; 
+        scheduledInterviews.forEach((scheduledInterview) => {
+          scheduledInterviewsSection.appendChild(createScheduledInterviewCard(email, scheduledInterview)); 
+        })
+      }
+    }); 
+}
+
+// Creates a scheduledInterview card using the fields of a scheduledInterview object
+function createScheduledInterviewCard(email, scheduledInterview) {
+  // Check the role of the current user for the scheduledInterview
+  let role;  
+  if(email === scheduledInterview.interviewerEmail) {
+    role = 'Interviewer'; 
+  } else {
+    role = 'Interviewee';
+  }
+  const scheduledInterviewElement = document.createElement('div');
+  scheduledInterviewElement.className = 'row';
+
+  const scheduledInterviewCard = document.createElement('div');
+  scheduledInterviewCard.className = 'card w-75 scheduled-interview-card';
+
+  const scheduledInterviewCardBody = document.createElement('div');
+  scheduledInterviewCardBody.className = 'card-body';
+
+  const scheduledInterviewCardTitle = document.createElement('h5');
+  scheduledInterviewCardTitle.className = 'card-title'; 
+  scheduledInterviewCardTitle.innerText = `Your role: ${role}`;
+
+  const scheduledInterviewCardText = document.createElement('p');
+  scheduledInterviewCardText.className = 'card-text'; 
+  scheduledInterviewCardText.innerText = createDateString(scheduledInterview); 
+
+  const scheduledInterviewCardAttendees = document.createElement('ul'); 
+  scheduledInterviewCardAttendees.className = 'list-group list-group-flush'; 
+
+  const scheduledInterviewCardInterviewee = document.createElement('li'); 
+  scheduledInterviewCardInterviewee.className = 'list-group-item'; 
+  scheduledInterviewCardInterviewee.innerText = `Interviewee Email: ${scheduledInterview.intervieweeEmail}`; 
+
+  const scheduledInterviewCardInterviewer = document.createElement('li'); 
+  scheduledInterviewCardInterviewer.className = 'list-group-item'; 
+  scheduledInterviewCardInterviewer.innerText = `Interviewer Email: ${scheduledInterview.interviewerEmail}`;
+
+  scheduledInterviewCardAttendees.appendChild(scheduledInterviewCardInterviewee); 
+  scheduledInterviewCardAttendees.appendChild(scheduledInterviewCardInterviewer); 
+  scheduledInterviewCardBody.appendChild(scheduledInterviewCardTitle); 
+  scheduledInterviewCardBody.appendChild(scheduledInterviewCardText); 
+  scheduledInterviewCard.appendChild(scheduledInterviewCardBody); 
+  scheduledInterviewCard.appendChild(scheduledInterviewCardAttendees);
+  scheduledInterviewElement.appendChild(scheduledInterviewCard); 
+  return scheduledInterviewElement; 
+}
+
+function isEmptyObject(obj) {
+  if (obj.length && obj.length > 0) {
+      return false;          
+  }
+  if (obj.length === 0) {
+     return true;       
+  }    
```

Comment:
Why do we need all these checks? 
Anyways, check on this: https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object
because it's probably tested better.